api/reporting/schemas/input/excluded_columns/user.txt

```diff
@@ -1,9 +1,11 @@
-version
-id_verification_is_valid
-cluster_create_retries
 billing_project_retries
```

Comment:
The basic rule for what to exclude is: is the information surfaced in a user-facing area of the product. If not, it doesn't really belong in the reporting dataset.

The user and workspace tables cover maybe 80% of what's being asked for, so I'm starting with those and punting on the rest for now.